Saturation/Splitter.cpp

```diff
@@ -1497,6 +1497,16 @@ void Splitter::onClauseReduction(Clause* cl, ClauseIterator premises, Clause* re
   }
 }
 
+void Splitter::addConditionalReduction(SplitSet* splits, ConditionalRedundancyEntry* e)
```

Comment:
Not to split hairs, but "conditionally reduced" already has a meaning in AVATAR. This is a different meaning, right?

Reply:
Then maybe just `addConditionalRedundancyEntry`? The concepts are indeed very similar, not just in name. 😃